content/coordination/onboarding/index.en.md

```diff
@@ -18,7 +18,7 @@ timely.
     Team, the messages to this account are forwarded to the email
     address you provided during your own onboarding process.
 
--   On the rladies.org website, we ask any prospective organisers to get
+-   On the rladies.org website, we ask any prospective organizers to get
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.